safe/gui/tools/help_dialog.py

```diff
@@ -33,7 +33,9 @@ def __init__(self, parent=None, message=None):
         :type parent: QWidget
         """
 
-        QtGui.QDialog.__init__(self, parent)
+        QtGui.QDialog.__init__(
+            self, parent,
+            flags=Qt.WindowMinimizeButtonHint | Qt.WindowMaximizeButtonHint)
         self.setupUi(self)
         self.setWindowTitle(self.tr('InaSAFE %s Help' % get_version()))
```

Comment:
@Gustry yeah I saw that too - I'm happy to leave the nicer title in the ui file anyway....better than 'Dialog'